usr/lib/common/loadsave.c

```diff
@@ -1749,18 +1816,37 @@ static CK_RV aes_256_unwrap(unsigned char key[32],
 
     memcpy(key, buffer, 32);
     rc = CKR_OK;
+
+    if ((tokdata->statistics->flags & STATISTICS_FLAG_COUNT_INTERNAL) != 0)
+        tokdata->statistics->increment_func(tokdata->statistics,
+                                            tokdata->slot_id,
+                                            &tokdata->store_strength.wrap_crypt,
+                                            tokdata->store_strength.wrap_strength);
+
 done:
     EVP_CIPHER_CTX_free(ctx);
     return rc;
 }
 
 CK_RV generate_master_key(STDLL_TokData_t *tokdata, CK_BYTE *key)
 {
+    CK_RV rc;
+
     if (tokdata->version < TOK_NEW_DATA_STORE)
         return generate_master_key_old(tokdata, key);
 
     /* generate a 256-bit AES key */
-    return rng_generate(tokdata, key, 32);
+    rc = rng_generate(tokdata, key, 32);
+
```

Comment:
When you are at it: there are constants in common/defs.h for the various key lengths:
```
#define AES_KEY_SIZE_256 32
#define AES_KEY_SIZE_192 24
#define AES_KEY_SIZE_128 16
```
So we could use AES_KEY_SIZE_256 here, instead of 32. But I see, this was old code.